app_dart/lib/src/service/github_service.dart

```diff
@@ -15,27 +15,40 @@ class GithubService {
   final GitHub github;
 
   /// Lists the commits of the provided repository [slug] and [branch].
-  Future<List<RepositoryCommit>> listCommits(
-      RepositorySlug slug, String branch, int hours, Config config) async {
-    final Duration time = Duration(hours: hours);
+  Future<List<RepositoryCommit>> listCommits(RepositorySlug slug, String branch,
+      int lastCommitTimestampMills, Config config) async {
     ArgumentError.checkNotNull(slug);
     final PaginationHelper paginationHelper = PaginationHelper(github);
 
+    /// Return only one page when this is a new branch. Otherwise it will
```

Comment:
maybe expand a little bit on what a page means in this context?

Reply:
Added doc: `The [pages] defines the number of pages of returned http request results.`